src/utils/cql/evaluateCql.js

```diff
@@ -0,0 +1,75 @@
+const { Client } = require('cql-translation-service-client');
+const { Library, Executor, Repository } = require('cql-execution');
+const { PatientSource } = require('cql-exec-fhir');
+const atob = require('atob');
+const axios = require('axios');
+const { ValueSetLoader } = require('./ValueSetLoader');
+const { LIBRARYS } = require('../../storage/collections');
+const debug = require('../../storage/logs').debug('medmorph-backend:evaluateCql');
+const db = require('../../storage/DataAccess');
+const configUtil = require('../../storage/configUtil');
+
+async function evaluateCQL(resources, expression, library, patientId) {
+  const elm = library.elm || (await fetchELM(library));
+  const valueSetLoader = new ValueSetLoader(library);
+  const valueSetMap = await valueSetLoader.seedValueSets();
+  const result = executeElm(resources, elm, valueSetMap);
```

Comment:
We lost the sample PlanDef on the internal FHIR server, so testing is a little tough at the moment, but is this where `valueSetMap` should be used? In `executeElm` below the third param is for dependent libraries, so like if we were using FHIRHelpers or something it would go in there

Reply:
I pushed the library back up. I left out a small piece in `executeElm` that I just pushed up. This seems to be where the `abstraction-tool` used the `valueSetMap`